### PR TITLE
Adding README capz upstream command

### PR DESCRIPTION
**Pull request by knabben**

### CODE REVIEW

**Comment by marosset** [comment]
Hmm, looking at the logs for `operational-tests-capz-windows-2019` it looks like the cluster came online but we didn't run ./hack/run-tests.sh from the windows-operational-readiness repo..
Looking

**Comment by knabben** [comment]
@marosset we dont have the script in the repo. I will create the file on this PR

**Comment by marosset** [comment]
> @marosset we dont have the script in the repo. I will create the file on this PR

Ahhh :)

File: hack/run-tests.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+export KUBERNETES_VERSION="v1.25.0"
```
Comment (marosset) [inline hack/run-tests.sh]:
```suggestion
KUBERNETES_VERSION=${KUBERNETES_VERSION:-"v1.25.0"}
```

so we can take this as an env var

**Comment by marosset** [comment]
Left one small suggestion, everything else looks good

**Comment by knabben** [comment]
curiously this test is failing consistently, checking if this is running in another place

```
Kubernetes e2e suite: [It] [sig-network] Services should be able to up and down services
```

**Pull request merged**